### DISCUSSION TITLE
SURICATA logs are not being seen in the Elasticsearch index.

### DISCUSSION THREAD

**Discussion by 89sooner** [Security-Onion-Solutions/securityonion#10575] [Unsupported Versions]
Hi, I am currently building an IDS with securityonion. We currently have a distributed architecture and after rebooting 1 of the 4 forward nodes, I am getting no eve logs, but after ifdown->ifup the monitor interface on the forward node, the /nsm/logs/suricata/eve* logs are accumulating well.
 
However, my guess is that the filebeat traffic is not being sent well from the forward node to the manager node.

I tried to capture 'port 5644' with tcp dump from hsnc-zd-sensor01(forward) node which is not working properly and I can see the following packets.
![is_not_ok](https://github.com/Security-Onion-Solutions/securityonion/assets/16891878/04095cd7-2afe-4be7-8e42-f61b76b6a281)

When I captured the same tcp dump with 'port 5644' on the forward node that is working normally, it seems that there is normal communication between hsnc-zd-sensor04(forward) and hsnc-zd-mgr(manager) nodes.
![is_ok](https://github.com/Security-Onion-Solutions/securityonion/assets/16891878/c77178ad-e0d8-4a36-b084-df444aaebaa1)

Finally, the suricata alert index is not resolving in elasticsearch on the search node.

Please help, I really want to know the solution.

**Comment by InfosecGoon** [comment]
Is there anything helpful in the logstash logs on the Manager? /opt/so/log/logstash/logstash.log

**Reply by jrm360seclab** [reply]
I am having the same problem but there is nothing I can see in my /opt/so/log/logstash/logstash.log pointing to my forward node, any other ideas?

**Reply by 89sooner** [reply]
I don't see anything unusual in logstash.log either, is there anything else?

**Reply by ben-sec** [reply]
Did you check your Elasticsearch logs as well (Manager and Search Nodes)? I had a few times the problem, that I reached the limit of 1.000 Elastic shards and Elastic stopped to index logs (starting with Suricate for whatever reason ...).

**Reply by jrm360seclab** [reply]
Thank you @ben-sec , I checked but only noticed certificate errors but nothing pointing to the forward node.

**Reply by jrm360seclab** [reply]
I was able to solve the issue for me hopefully this will help you too. find the answer there.

https://github.com/Security-Onion-Solutions/securityonion/discussions/10607
